src/librustdoc/html/render/context.rs

```diff
@@ -602,13 +604,17 @@ impl<'tcx> FormatRenderer<'tcx> for Context<'tcx> {
         let mut sidebar = Buffer::html();
         write!(sidebar, "<h2 class=\"location\"><a href=\"#\">Crate {}</a></h2>", crate_name);
```

Comment:
One last thing: This `<h2>` should be rendered by the template instead. You can remove this `write!` and set `title_prefix` and `title` appropriately.